include/internal.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2018-2022 Yubico AB. All rights reserved.
+ * Copyright (c) 2022 Felix Gohla, Konrad Hanff, Tobias Kantusch,
+ *                    Quentin Kuth, Felix Roth. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file.
+ */
+
+#pragma once
+
+#include <stdint.h>
+
+#include "dev.h"
+
+int fido_rx(fido_dev_t *, const uint8_t, void *, const size_t);
```

Comment:
I suggest adding the parameter names to all functions in this file.

Reply:
Should be added to the headers.